kickoff/static/suggestions.js

```diff
@@ -255,45 +255,60 @@ function getElmoUrl(productName, version) {
     return url;
 }
 
-function getPreviousBuildL10nUrl(productName, version, previousBuildNumber) {
+function _getL10nChangesetsOptsOfNewVersion(productName, version, branchName, revision) {
+    var opts = {};
+    if (productName === 'thunderbird' || version < '59') {
+        opts.url = getElmoUrl(productName, version);
+        opts.downloadPlaceholder = 'Elmo';
+    } else {
+        opts.downloadPlaceholder = 'l10n-changesets.json on hg.mozilla.org';
+        var topFolder = productName === 'fennec' ? 'mobile' : 'browser';
+        opts.url = getInTreeFileUrl(branchName, revision, topFolder + '/locales/l10n-changesets.json');
+    }
+
+    opts.previousBuildWarning = '';
+    return opts;
+}
+
+function _getPreviousBuildL10nUrl(productName, version, previousBuildNumber) {
     var baseUrl = window.location.origin;
     var releaseFullName = [productName, version, 'build' + previousBuildNumber].join('-');
     return baseUrl + '/releases/' + releaseFullName + '/l10n';
 }
 
-function getUrlAndMessages(productName, version, buildNumber) {
-    var opts = {
-        url: '',
-        downloadPlaceholder: 'Trying to download from ',
-        previousBuildWarning: '',
-    };
-    var minor = version.match(/(\d+)\.\d+\.\d+/);
-    if (minor) {
+function _getL10nChangesetsOptsOfPreviousBuild(productName, version, buildNumber, minorVersionNumber) {
+    var opts = {};
+    if (minorVersionNumber) {
         // Fall back to $major.0 build1
         var previousBuildNumber = 1;
-        var previousVersion = minor[1] + '.0';
+        var previousVersion = minorVersionNumber[1] + '.0';
```

Comment:
I see you're moving old code, but ... if we take an l10n fix in x.0.1, this snippet means we use x.0 build1 changesets for x.0.2, and regress the fix yes ? Or if we changed l10n between x.0 builds. This are pretty rare but perhaps it's worth filing this for another time.

Reply:
Good point. Let me see how hard it is to fix it here.

Reply:
Fixed